.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Why adding this?

Reply:
Its a first step towards making this repo compatible with Black's default options (this change is done in MQT already).

Reply:
But the file needs to end with an empty line, right?